rules/manager.go

```diff
@@ -37,9 +37,9 @@ import (
 const (
 	namespace = "prometheus"
 
-	ruleTypeLabel     = "rule_type"
-	alertingRuleType  = "alerting"
-	recordingRuleType = "recording"
+	RuleTypeLabel     = "rule_type"
```

Comment:
On Tue, May 26, 2015 at 12:29 PM, Fabian Reinartz
<notifications@github.com> wrote:
>
> Do we agree that it's a bit weird to export AlertingRule but not recordingRule?

Yes, I do find that weird. Even if recordingRule is not used outside
of the package at the moment.


-- 
Björn Rabenstein, Engineer
http://soundcloud.com/brabenstein

SoundCloud Ltd. | Rheinsberger Str. 76/77, 10115 Berlin, Germany
Managing Director: Alexander Ljung | Incorporated in England & Wales
with Company No. 6343600 | Local Branch Office | AG Charlottenburg  |
HRB 110657B